client/post-editor/media-modal/index.jsx

```diff
@@ -2,13 +2,10 @@
  * External dependencies
  */
 var React = require( 'react' ),
-	noop = require( 'lodash/noop' ),
-	head = require( 'lodash/head' ),
-	some = require( 'lodash/some' ),
-	findIndex = require( 'lodash/findIndex' ),
-	values = require( 'lodash/values' ),
 	closest = require( 'component-closest' ),
 	debug = require( 'debug' )( 'calypso:post-editor:media' );
+import { connect } from 'react-redux';
+import { noop, head, some, findIndex, partial } from 'lodash';
```

Comment:
>alphabetize much? :wink:

Curious what benefit there'd be in alphabetizing it? To me it seems an unlikely case that we'd be looking for any one particular item in the set of imports. [`no-redeclare`](http://eslint.org/docs/rules/no-redeclare) would avoid accidentally importing something twice and [`no-undef`](http://eslint.org/docs/rules/no-undef) would avoid calling on a function that's not imported (assuming we're unaware whether it exists in scope). Granted, these require some editor integration to make the issue immediately apparent, though I'd argue we should promote that anyways.

It's a small point to argue, but seems like we'd have a hard time enforcing it, and the time we'd spend trying to enforce it aren't for much benefit.